logos/logos-orchestrator/src/logos/main.py

```
@@ -769,12 +770,13 @@ def _capture_logosnode_provider_snapshot(
                         _resolve_provider_name(provider_id),
                         exc_info=True,
                     )
+        return snapshot_id
 
-    sample["snapshot_id"] = snapshot_id
+    sample["snapshot_id"] = await with_db(_persist)
     asyncio.create_task(_logosnode_registry.record_runtime_sample(provider_id, sample))
```

Ruff (0.15.15)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)